src/fides/api/models/privacy_preference.py

```diff
@@ -155,7 +154,7 @@ class PrivacyPreferenceHistory(Base):
         ),
     )
 
-    user_geography = Column(EnumColumn(PrivacyNoticeRegion), index=True)
+    user_geography = Column(String, index=True)
```

Comment:
This is already a character varying column at the db level so no db migration needed